Ginger/GingerCore/ALM/JIRA/Bll/JiraConnectManager.cs

```diff
@@ -96,7 +100,9 @@ private bool JiraConnectionTest()
             if (jiraDomainsProjectsDataList.Count > 0)
             {
                 currentDomainProject = jiraDomainsProjectsDataList.Where(dom => dom.DomainName.Equals(ALMCore.DefaultAlmConfig.ALMDomain)).Select(prj => prj.Projects).FirstOrDefault();
-                jiraProjects = currentDomainProject.ToDictionary(x => x.Prefix, x => x.ProjectName);
+                // olego 9/29/20 to be sure that nothing not impacted by change
+                // jiraProjects = currentDomainProject.ToDictionary(x => x.Prefix, x => x.ProjectName);                     
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)